scripts/apps/authoring-react/article-widgets/versions-and-item-history/versions-tab.tsx

```diff
@@ -114,11 +114,11 @@ export class VersionsTab extends React.PureComponent<IArticleSideWidgetComponent
 
         compareAuthoringEntities({
             item1: {
-                label: gettext('version {{n}}', {n: from._current_version}),
+                label: gettext('version {{n}}', {n: from?._current_version}),
```

Comment:
You can't output a string with pieces missing to the UI - which would happen since you use optional chaining without a fallback.

Current version is mandatory, not sure why are you applying optional chaining all over the place here.